04week/nasa/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <link rel="stylesheet" href="./css/style.css">
+  <link href="https://fonts.googleapis.com/css?family=Noto+Sans" rel="stylesheet">
+</head>
+
+<body>
+  <span class="nasa"> <h1 class="text">NASA</h1>
+</span>
+  <!-- main section with 3 divs -->
+  <main>
+    <div class="social">
+      <h2>Social Media</h2>
+      <p>
+        Twitter- @nasa <br /> Facebook- fb.com/nasa <br /> Instagram- @nasa <br /> Youtube- youtube.com/NASA
+      </p>
+    </div>
+    <div class="office">
+      <h2>OFFICE INFO</h2>
+      <p>
+        Public Communications Office <br /> NASA Headquarters <br /> 300 E. Street SW, Suite 5R30 <br /> Washington, DC 20546 <br /> (202) 358-001 (Office) <br /> (202) 358-4338 (Fax)
+      </p>
+    </div>
+    <div class="contact">
+      <h2>CONTACT US</h2>
+      <form>
+        Name:<br>
+        <input type="text" name="firstname" class="name"><br> Message:
+        <br>
+        <input type="text" name="message" class="message">
+        <input type="submit" class="send" value="send">
```

Comment:
you need some space between the different inputs, they are all scrunched together.